pkgs/data/themes/flat-remix-gnome/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, fetchFromGitHub
+, glib
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "flat-remix-gnome";
+  version = "20210623";
+
+  src = fetchFromGitHub {
+    owner = "daniruiz";
+    repo = pname;
+    rev = version;
+    hash = "sha256-FKsbAvjhXb2ipe3XqACM6OwGYwbBbzvDjsUQYCIQ8NM=";
+  };
+
+  nativeBuildInputs = [ glib ];
+  makeFlags = [ "PREFIX=$(out)" ];
+  preInstall = ''
+    # https://github.com/daniruiz/flat-remix-gnome/blob/20210623/Makefile#L50
+    mkdir -p $out/share/gnome-shell/
+    touch $out/share/gnome-shell/gnome-shell-theme.gresource
```

Comment:
Why touch it, though?

Reply:
Please check out the code of comment, `make install` step will backup file `gnome-shell-theme.gresource`.
If the file not exists, `make install` will fail. So I touch a fake gresource and delete the backup file.

Reply:
Oh, right. I was reading the wrong branch. An explanation comment might still be useful.

Reply:
Explanation comment added : )